docs/resources.md

```diff
@@ -162,6 +162,60 @@ spec:
       value: gcr.io/staging-images/kritis
 ```
 
+#### Surfacing the image digest built in a task
+
+To surface the image digest in the output of the `taskRun` the builder tool should produce this information in a [oci-layout-image](https://github.com/opencontainers/image-spec/blob/master/image-layout.md) `index.json` file. This file should be placed on a location as specified in the task definition under the resource `outputImagePath`
+
+for example this build-push task defines the `outputImagePath` for the `buildImage` resource in `/worksapce/buildImage`
+```yaml
+apiVersion: tekton.dev/v1alpha1
+kind: Task
+metadata:
+  name: build-push
+spec:
+  inputs:
+    resources:
+    - name: workspace
+      type: git
+  outputs:
+    resources:
+    - name: builtImage
+      type: image
+      outputImagePath: /workspace/builtImage
+  steps: ...
+```
+If no value is specified for `outputImagePath`, it will default to `/tools/image-outputs/{resource-name}`.
+
+*Please check the builder tool used on how to pass this path to create the output file.*  
+
+The `taskRun` will include the image digest in the `resourcesResult` field that is part of the `taskRun.Status` 
+
+for example:
+
+```yaml
+status:
+    completionTime: 2019-04-29T14:20:41Z
+    conditions:
+    - lastTransitionTime: 2019-04-29T14:20:41Z
+      status: "True"
+      type: Succeeded
+    podName: build-push-run-pod-0a1777
+    resourcesResult:
+    - digest: sha256:eed29cd0b6feeb1a92bc3c4f977fd203c63b376a638731c88cacefe3adb1c660
+      name: skaffold-image-leeroy-web
+    startTime: 2019-04-29T14:20:30Z
+    steps:
+    - name: build-and-push
+      terminated:
+        containerID: docker://140fbe0d923bd0196e0ef1edbf088731ff1e8d37bfa9b04307701ccbdae77426
+        exitCode: 0
+        finishedAt: 2019-04-29T14:20:39Z
+        reason: Completed
+        startedAt: 2019-04-29T14:20:36Z
```

Comment:
i think you could remove the non-relevant sections from `status` without losing anything important, e.g.:

```
status:
...
    resourcesResult:
    - digest: sha256:eed29cd0b6feeb1a92bc3c4f977fd203c63b376a638731c88cacefe3adb1c660
      name: skaffold-image-leeroy-web
...
```